docs/options.md

```diff
@@ -395,6 +397,22 @@ Type: `string | "current" | true`.
 
 If you want to compile against the current node version, you can specify `"node": true` or `"node": "current"`, which would be the same as `"node": process.versions.node`.
 
+Alternatively, you can specify node version in a browserslist query:
+
+```jsonc
+{
+  "targets": "node 12" // not recommended
+}
+```
+
+In this case, browserslist will interpret it as `^12.0.0`, which is resolved to
```

Comment:
I find this confusing, because to me `^12.0.0` means "I want to support every version starting from 12.0.0. I think we should just say "browserslist will resolve it to the _latest_ version available", without mentioning `^12.0.0`.